packages/zcli-apps/src/lib/buildAppJSON.ts

```diff
@@ -135,7 +135,7 @@ export const buildAppJSON = async (appPaths: string[], port: number): Promise<Ap
     const manifest = getManifestFile(appPath)
     const zcliConfigFile = getAllConfigs(appPath) || {}
 
-    const appId = zcliConfigFile.app_id?.toString() || uuidV4()
+    const appId = zcliConfigFile.app_id?.toString() || '0'
```

Comment:
Just curious: does it matter if it's string or integer? According to https://developer.zendesk.com/api-reference/ticketing/apps/apps/#send-notification-to-app it's integer.

Reply:
@Frederick888 Yes this documentation is for the notify endpoint and yes it requires `app_id` as an integer. But within ZCLI it doesn't have to be integer and the preference was already for string so I decided to follow it.

Reply:
Cool, as long as it works it shouldn't matter too much 👍